extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/test/java/io/quarkus/resteasy/reactive/server/test/resource/basic/SubResourceRequestFilterTest.java

```diff
@@ -59,8 +61,8 @@ public RestSubResource hello() {
     public static class RestSubResource {
 
         @GET
-        public Response hello(HttpHeaders headers) {
-            return Response.ok().header("single-filter", headers.getHeaderString("single-filter")).build();
+        public Response hello(HttpHeaders headers, @RestPath String name) {
```

Comment:
Here, though, I do wonder if `@RestPath` is required. I suppose yes, because there's no `@Path` matching. I am not sure what the spec says about sub-resources allowing `@Path` matching. Is it allowed/supported?

Reply:
Yeah, here it is required because there is no other way to know